frontend/src/game/NetworkService.ts

```
@@ -0,0 +1,168 @@
+
+          this.socket.onmessage = (event) => {
+            if (this.GameState) {
+              try {
+                const uint8Array = new Uint8Array(event.data)
+                const message = this.GameState.decode(uint8Array)
+                const state = this.GameState.toObject(message, {
+                  defaults: true,
+                  arrays: true,
+                  objects: true,
+                })
+                this.listeners.forEach((callback) => callback(state))
```

Biome (2.4.6)

[error] This callback passed to forEach() iterable method should not return a value.

(lint/suspicious/useIterableCallbackReturn)